src/main/java/kr/codesqaud/cafe/controller/MemberController.java

```diff
@@ -48,12 +51,14 @@ public String getUser(@PathVariable String userId, Model model) {
     }
 
     @GetMapping("/users/{userId}/form")
-    public String updateUser(@PathVariable("userId") String userId, Model model) {
+    public String updateUser(@PathVariable("userId") String userId, Model model,HttpSession session) {
         logger.debug("updateUser : GET");
-
-        Optional<Member> updateUser = jdbcMemberRepository.findById(userId);
-        // Model 과 View 연결
-        model.addAttribute("user", updateUser.orElseThrow());
+        Object value = session.getAttribute("sessionedUser");
+        if(value != null){
+            Optional<Member> updateUser = jdbcMemberRepository.findById(userId);
```

Comment:
```suggestion
            Member updateUser = jdbcMemberRepository.findById(userId).orElseThrow();
```

value == null 이면 어떻게 되는거죠? 뭔가 로직이 살짝 모호해 보입니다.